mPower2/MotorControl/ViewControllers/MCTTappingStepViewController.swift

```diff
@@ -121,10 +116,16 @@ public class MCTTappingStepViewController: RSDActiveStepViewController, MCTHandS
         self.nextButton?.isHidden = true
     }
     
+    /// Override view will appear to set the unit label text
+    override public func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.unitLabel?.text = "TAP COUNT" // TODO rkolmos 04/11/2018 localize
```

Comment:
`Localization.localizedString("TAP_COUNT_LABEL")` and then in the strings file add:

`"TAP_COUNT_LABEL" = "TAP COUNT";

Before we were adding the localization TODO comment b/c the strings file hadn't been added.